deepwell/src/services/special_page/service.rs

```diff
@@ -179,7 +181,10 @@ impl SpecialPageService {
         // a desired localization property of Fluent.
         //
         // See https://fluent-compiler.readthedocs.io/en/latest/usage.html#:~:text=You%20will%20notice%20the%20extra%20characters%20\u2068%20and%20\u2069%20in%20the%20output.
-        char_replace_in_place(&mut wikitext, &['\u{2068}', '\u{2069}'], "");
+        static CONTROL_CHAR_REGEX: LazyLock<Regex> =
+            LazyLock::new(|| Regex::new("[\u{2068}|\u{2069}]").unwrap());
```

Comment:
I think this square bracket regex also replaces the pipe character?

Reply:
Oh you're right. A dash would've been fine or not having a char group but this isn't okay. Damn, good catch.